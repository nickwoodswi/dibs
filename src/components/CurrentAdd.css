.current-add{
    width: 100%;
    text-align: center;
    font-size: 16px;
    margin-top: 15px;
}