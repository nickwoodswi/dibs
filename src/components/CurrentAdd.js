import React, { Component } from 'react'
import './CurrentAdd.css'

class CurrentAdd extends Component {
    static defaultProps = {
        appState: {}
    }
    render() {
        return(
        <div className="current-add">
            <div className="current-add-header">CURRENTLY ADDING HOLDS FROM:</div>
            {this.props.appState.holdmonth_start} {this.props.appState.holdday_start}, {this.props.appState.holdyear_start} thru {this.props.appState.holdmonth_end} {this.props.appState.holdday_end}, {this.props.appState.holdyear_end}
        </div>
        )
    }
}

export default CurrentAdd