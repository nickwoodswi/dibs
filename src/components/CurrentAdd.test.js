import React from 'react';
import { render } from '@testing-library/react';
import CurrentAdd from './CurrentAdd';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

test('renders current add component', () => {
    render(<MemoryRouter><CurrentAdd /></MemoryRouter>);
});

it('CurrentAdd renders the UI as expected', () => {
    const tree = renderer
        .create(<MemoryRouter><CurrentAdd /></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});