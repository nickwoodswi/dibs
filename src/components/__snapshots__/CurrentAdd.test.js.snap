// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentAdd renders the UI as expected 1`] = `
<div
  className="current-add"
>
  <div
    className="current-add-header"
  >
    CURRENTLY ADDING HOLDS FROM:
  </div>
   
  , 
   thru 
   
  , 
</div>
`;
